python/scripts/rfi_receiver/rfi_receiver.py

```diff
@@ -162,14 +167,7 @@ def __init__(self):
                     argument.config
                 )
             )
-            # TODO: this was disabled because it ignores the structure of the kotekan
-            # config, looks in the whole kotekan config tree for the keys from the
-            # config hardcoded in this file. It ignores that the kotekan config has
-            # variable names reused in different parts of the config and just applies
-            # all of them, resulting in wrong values, i.e. for samples_per_data_set.
-            # To solve this issue, only the hardcoded config is used, but it should
-            # become a config file or fin d a better way to read the kotekan config.
-            # parse_dict(self, yaml.safe_load(open(argument.config)))
+            parse_dict(self, yaml.safe_load(open(argument.config)))
```

Comment:
Since we are pushing the conf options to the file, I think we should remove them from the source file, see line 77

Reply:
The `config` struct is needed as part of the `CommandLine` class